src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -591,13 +591,13 @@ public partial class ExtendedProtectionPolicy : System.Runtime.Serialization.ISe
         public ExtendedProtectionPolicy(System.Security.Authentication.ExtendedProtection.PolicyEnforcement policyEnforcement) { }
         public ExtendedProtectionPolicy(System.Security.Authentication.ExtendedProtection.PolicyEnforcement policyEnforcement, System.Security.Authentication.ExtendedProtection.ChannelBinding customChannelBinding) { }
         public ExtendedProtectionPolicy(System.Security.Authentication.ExtendedProtection.PolicyEnforcement policyEnforcement, System.Security.Authentication.ExtendedProtection.ProtectionScenario protectionScenario, System.Collections.ICollection customServiceNames) { }
```

Comment:
`System.Collections.ICollection customServiceNames`, the parameter appears to allow null.

[https://github.com/dotnet/runtime/blob/8ddc3e11454b1f4cd4df27003d1bbfbb1212e9d1/src/libraries/System.Net.Security/src/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicy.cs#L42-L48](https://github.com/dotnet/runtime/blob/8ddc3e11454b1f4cd4df27003d1bbfbb1212e9d1/src/libraries/System.Net.Security/src/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicy.cs#L42-L48)